### PR TITLE
added global reach flag in New-AzureExpressRouteCircuit cmdlet 

### PR DESCRIPTION
**Pull request by syfarogh**
## Description

added global reach flag in New-AzureExpressRouteCircuit cmdlet to toggle Global reach feature on a circuit.

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs [hunk 1]
```diff
@@ -103,6 +103,11 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool? AllowClassicOperations { get; set; }
 
+        [Parameter(
+           Mandatory = false,
+           ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNullOrEmpty]
+        public bool AllowGlobalReach { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: Boolean parameters are discouraged.  You should use a SwitchParameter instead.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs [hunk 2]
```diff
@@ -103,6 +103,11 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool? AllowClassicOperations { get; set; }
 
+        [Parameter(
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: You should ensure the new parameter is covered in one of your tests

**Comment by markcowl** [comment]
@syfarogh Code complete is today (8/6) Please make requested changes by EOD if you would like this PR in this release.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs (hunk 2 above)
Comment (syfarogh) [reply src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: I've updated an existing test to test this flag.

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.ps1 [hunk 3]
```diff
@@ -154,7 +154,7 @@ function Test-ExpressRouteCircuitCRUD
       $resourceGroup = New-AzureRmResourceGroup -Name $rgname -Location $rglocation
       
       # Create the ExpressRouteCircuit
-		$circuit = New-AzureRmExpressRouteCircuit -Name $circuitName -Location $location -ResourceGroupName $rgname -SkuTier Standard -SkuFamily MeteredData  -ServiceProviderName "equinix" -PeeringLocation "Silicon Valley" -BandwidthInMbps 500;
+		$circuit = New-AzureRmExpressRouteCircuit -Name $circuitName -Location $location -ResourceGroupName $rgname -SkuTier Standard -SkuFamily MeteredData  -ServiceProviderName "equinix" -PeeringLocation "Silicon Valley" -BandwidthInMbps 500 -AllowGlobalReach;
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.ps1]: Looks like you will need to re-record tests for this

**Comment by maddieclayton** [comment]
@syfarogh Ping

**Comment by maddieclayton** [comment]
Closing per offline conversation that @syfarogh will open a new PR targeting a different branch.

**Pull request closed without merging**